pkg/proto/nbcontract/v1/identityconfig.proto

```diff
@@ -2,16 +2,8 @@ syntax = "proto3";
 package nbcontract.v1;
 
 message IdentityConfig {
-  IdentityType identity_type = 1;
```

Comment:
Removing, since not seeing this used anywhere

Reply:
Makes sense to me too. If later on we have other identity type to support, we can add it back.